lib/sass_spec/engine_adapter.rb

```diff
@@ -66,20 +66,34 @@ def version
 
   def compile(sass_filename, style)
     require 'sass'
+    # restore old fh
+    stderr = StringIO.new
+    # restore previous default encoding
+    encoding = Encoding.default_external
+    # Does not work as expected when tests are run in parallel
+    # It runs tests in threads, so stderr is shared across them
+    old_stderr, $stderr = $stderr, stderr
```

Comment:
$stderr is global, how can make sure the output of parallel tests does not mix here?